content/docs/concepts/resources/functions.md

```diff
@@ -203,18 +356,18 @@ def get_kubeconfig(self,
 <div>
 <pulumi-choosable type="language" values="java">
 
-(no example available for Java)
+No example available for Java
```

Comment:
Are there just no examples in Java and YAML, or are provider methods not supported in those languages? If not supported, is it that they aren't supported _yet_ (in which case it'd also be good to link to an issue), or that they'll never be? (Generally best to be clear on things like this, as folks will likely wonder the same.)

Reply:
I'm not actually sure on this -- I've merely (perhaps wrongly) added some cosmetic changes to this bit but am not responsible for the original docs (nor sure on the answers to your questions).